ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -28,9 +29,10 @@ import (
 )
 
 const (
-	RayJobDefaultRequeueDuration    = 3 * time.Second
-	RayJobDefaultClusterSelectorKey = "ray.io/cluster"
-	PythonUnbufferedEnvVarName      = "PYTHONUNBUFFERED"
+	RayJobDefaultRequeueDuration        = 3 * time.Second
+	RayJobDefaultClusterSelectorKey     = "ray.io/cluster"
+	PythonUnbufferedEnvVarName          = "PYTHONUNBUFFERED"
+	DELETE_RAYJOB_CR_AFTER_JOB_FINISHES = "DELETE_RAYJOB_CR_AFTER_JOB_FINISHES"
```

Comment:
DeleteRayJobAfterJobFinishesEnvVar

Reply:
I am wondering when to use Pascal case and when to use Constant case. This file also use Constant case.
https://github.com/ray-project/kuberay/blob/ea314d7ee780e840382f3de852918f69d7867f25/ray-operator/controllers/ray/rayservice_controller.go#L45

Reply:
According to @kevin85421's suggestion below, I've moved this variable to `constant.go`.
https://github.com/ray-project/kuberay/pull/2225#discussion_r1671639314